doc/specification.md

```diff
@@ -75,3 +76,43 @@ To enable interchangeability the files are characterized by
 
 The JSON files additionally follow
 * 2 spaces indentation
+
+## Format Compatibility
+
+The format of object types will evolve over time. If a change to the format is considered incompatible, the version of the file format has to be increased.
+
+As of today, the format version is indicated by the `$schema` attribute. This is approach is in discussion and might be changed.
+
+### Compatible File Format Changes 
+
+For compatible changes to the file format, the format is just updated. The indicator whether the format has to be changed should not be updated.
+
+Following changes to file formats are considered as compatible : 
+
+- Descriptions or titles are changed
+- Values for a field (enum) are added (remark: this might lead to syntax errors in ABAP system that don't support this value)
```

Comment:
```suggestion
- Values for a field (enum) are added (remark: this might lead to syntax errors in ABAP system that doesn't support this value)
```

Reply:
Thanks, I have just changed `system` to `systems`.